ProjectSettings/TagManager.asset

```diff
@@ -18,8 +18,8 @@ TagManager:
   - 
   - Environment
   - 
-  - 
-  - 
+  - Interactable
```

Comment:
nice conflicts in this file are easily solvable if there are any. didnt realize it was literally just a text file for the most part